Sources/BraintreeCore/BTAPIPinnedCertificates.swift

```diff
@@ -8,9 +8,9 @@ import Foundation
 ///
 /// Do not update this file directly. If new certificates are needed, update the `certificates` directory
 /// in `braintree-ios-certificate-generation` and re-run the certificate generation script.
-@objcMembers public class BTAPIPinnedCertificates: NSObject {
+class BTAPIPinnedCertificates {
```

Comment:
This class is only used in Core. Checked the Drop-in and it's not being used there so safe to make it internal.